frontend/packages/console-shared/src/utils/storage-utils.ts

```diff
@@ -12,3 +12,6 @@ export const isCephProvisioner = (scProvisioner: string): boolean => {
     _.endsWith(scProvisioner, provisioner),
   );
 };
+
+// To check that the storage class isn't noobaa based
+export const filterStorageClasses = (sc) => !sc.provisioner.includes('noobaa.io/obc');
```

Comment:
I think you can use `StorageClass` type from `@console/internal/components/storage-class-form`

```suggestion
export const filterStorageClasses = (sc: StorageClass) => !sc.provisioner.includes('noobaa.io/obc');
```